lib/LaTeXML/Post/LexMath.pm

```diff
@@ -32,6 +32,8 @@ sub convertNode {
 sub rawIDSuffix {
   return '.lm'; }
 
+sub canConvert { return 1; }
+
```

Comment:
Use the OOPs, son :>
Let's assume that MathProcessors *can* convert, unless they claim otherwise.  So put this method on `LaTeXML::Post::MathProcessor`, and then it only needs to be overridden for `MathML::Content` and `OpenMath`.

Reply:
Sure, done.